components/brave_news/browser/channel_migrator.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_news/browser/channel_migrator.h"
+
+#include <string_view>
+
+#include "base/containers/fixed_flat_map.h"
+#include "base/strings/strcat.h"
+#include "brave/components/brave_news/common/pref_names.h"
+#include "components/prefs/scoped_user_pref_update.h"
+
+namespace brave_news {
+
+namespace {
+
+constexpr auto kMigrateChannels =
+    base::MakeFixedFlatMap<std::string_view, std::string_view>(
+        {{"Tech News", "Technology"},
+         {"Sport", "Sports"},
+         {"Celebrity News", "Celebrities"},
+         {"Entertainment news", "Celebrities"},
```

Comment:
will the channel matching be case sensitive? If so, should `news` be capitalized?

Reply:
Made these case insensitive